src/rhai_engine.rs

```diff
@@ -6,12 +6,19 @@ pub fn custom_rhai_engine_init() -> Engine {
     let mut rhai_engine = Engine::new();
 
     rhai_engine.register_fn("current_date", current_date);
+    rhai_engine.register_fn("print_line", print_line);
 
     rhai_engine
 }
 
-/* Allows access for current_date in rhai script to allow for filtering
-The specific date format is to allow for comparision inside rhai scripts */
+// Allows access for current_date in rhai script to allow for filtering
+// The specific date format is to allow for comparision inside rhai scripts
 fn current_date() -> String {
     Utc::now().to_rfc3339_opts(SecondsFormat::Secs, true)
 }
+
+// Allows for some debugging capability for rhai scripts by printing to stdout
+// instead of having to serially print output of steps on the webpage
+fn print_line(message: String) {
+    println!("{message}");
```

Comment:
super duper mega nit: given the intended usage, would this be better on stderr?  (i.e. `eprintln!`)  It really doesn't matter though!

Reply:
Good point. I'll make the change.